easycv/hooks/lr_update_hook.py

```diff
@@ -54,3 +55,29 @@ def get_lr(self, runner, base_lr):
             target_lr = self.min_lr
 
         return annealing_cos(base_lr, target_lr, progress / max_progress)
+
+
+@HOOKS.register_module()
+class CosineAnnealingWarmupByEpochLrUpdaterHook(CosineAnnealingLrUpdaterHook):
+
+    def before_train_iter(self, runner: 'runner.BaseRunner'):
```

Comment:
CosineAnnealingLrUpdaterHook already support by epoch， what is the diff?

Reply:
Because CosineAnnealingLrUpdaterHook only can update the lr by epoch after warmup. CosineAnnealingLrUpdaterHook can not update the lr by epoch during warmup. CosineAnnealingWarmupByEpochLrUpdaterHook can do this.